.github/workflows/self_hosted_integration_tests.yml

```diff
@@ -74,6 +74,10 @@ jobs:
         run: |
           source activate accelerate;
           pytest -sv tests/deepspeed
+      - name: Clean local files
+        if: always()
+        run: |
+          rm -rf accelerate transformers
 
       - name: Run transformers examples tests
         working-directory: transformers/
```

Comment:
quick look: this seems a bit odd, where we delete `transformers` but having `working-directory: transformers` ..?

Reply:
my 2 comments may apply to multiple places (if they make sense)

Reply:
Yes we now have non-persistent workers so this shouldn't be necessary.